### PR TITLE
Design mobile

### PR DESCRIPTION
**Pull request by Romano78**
Did most of the responsiveness, I just need to finish the navbar and do the carousel and do the english version. But you can merge this so far and see if it work in production :)

### CODE REVIEW

**Comment by kristynadierstein** [comment]
Ola, 

thank you very much! Sorry - I was busy until now today but looking at it now and good job! I am posting again some comments I have for the Desktop as well as Mobile now: 

Desktop: 
1. missing Navbar
Screenshot 2020-06-14 at 17 41 18

2. width of the paragraph PRODUKTY as per Figma
Screenshot 2020-06-14 at 17 42 38
vs
Screenshot 2020-06-14 at 17 43 22

Please check the font sizes as well as now we have different size of Body inside the PRODUKTY body and the cards body: 
Screenshot 2020-06-14 at 17 45 06
 vs
this in Figma: 
Screenshot 2020-06-14 at 17 44 57

Also please word-wrap property to wrap the text inside paragraphs properly if too long. 
3. CARDS:
- padding on top of the text, now there is no space, pls refer to Figma
- bottom line is missing once the text appear
-this text is not seen entirely
Screenshot 2020-06-14 at 17 47 39

4. O NAS section has too big font, pls refer to Figma, the spacing on top of the text between the line and the body does not correspond to Figma either

**Comment by kristynadierstein** [comment]
Hele I am not merging this version to master as I have limited amount of deploys per month on Netlify and there is still a lot to be done...so I will be waiting for more ready to deploy version and this one I will only merge to master if it is ok... thank you 🙏
